src/commands/bootstrap/is-folder-empty.ts

```diff
@@ -0,0 +1,72 @@
+// Code copied from `create-next-app`.
+
+import { lstatSync, readdirSync } from 'node:fs';
+import { join } from 'node:path';
+import picocolors from 'picocolors';
```

Comment:
Seeing as we already use `chalk`, let's maintain consistency and follow-up with a separate PR to replace it with `picocolors` if we feel we'd benefit from it?

Wdyt?